vespa-dependencies-enforcer/allowed-maven-dependencies.txt

```diff
@@ -30,6 +30,8 @@ com.google.jimfs:jimfs:${jimfs.vespa.version}
 com.google.protobuf:protobuf-java:${protobuf.vespa.version}
 com.ibm.icu:icu4j:${icu4j.vespa.version}
 com.microsoft.onnxruntime:onnxruntime:${onnxruntime.vespa.version}
+de.kherud:llama:${kherud.llama.vespa.version}
+org.jetbrains:annotations:24.0.1
```

Comment:
Please exclude jetbrains annotations. It's purely a compile time dependency - the annotation are not required at runtime.